tests/test_type_descriptors.py

```diff
@@ -92,3 +111,27 @@ def test_bullet_point_descriptions(node: Object, description: str) -> None:
 def test_typescript_description(node: Object, description: str) -> None:
     """Verify typescript descriptions."""
     assert TypeScriptDescriptor().describe(node) == description
+
+
+@pytest.mark.parametrize(
+    "node,description",
+    [
+        (
+            OBJ1,
+            (
+                "```Postgres\n"
+                "\n"
+                "object: { // Object Description\n"
```

Comment:
The schema looks like it's something between typescript and postgres. I don't know how much this stuff matters since it's really hard to know what the LLM will do.

I was originally thinking about getting the postgres type description to output either the `CREATE TABLE` or a markdown table that shows the column/types/description of the table.

What are your thoughts on this?

```sql

CREATE TABLE sample_table (
    id SERIAL PRIMARY KEY,
    text_field TEXT,
    text_array_field TEXT[],
    float_field FLOAT,
    integer_field INTEGER,
    integer_array_field INTEGER[],
    embedded_json_field JSONB,
    color_enum_field COLOR_ENUM
);

CREATE TYPE color_enum AS ENUM ('RED', 'GREEN', 'BLUE');
```

or else the output from running:

```sql
SELECT column_name, data_type FROM information_schema.columns WHERE table_name = '...';   
```

something of this sort: 

```markdown
 column_name |     data_type
-------------+-------------------
 id          | integer
 text_field  | text
 text_array_field | text[]
 float_field | double precision
 integer_field | integer
 integer_array_field | integer[]
 embedded_json_field | jsonb
 color_enum_field | color_enum
```

Reply:
@eyurtsev You are right. I was still determining the exact syntax; the markdown syntax table looks correct. I will update my PR.

Reply:
One issue with the above approach is displaying nested objects. What would be the correct way to display nested objects in the markdown table format?